### PR TITLE
Make docs links relative

### PR DESCRIPTION
**Pull request by TerryHowe**
If we are going to support docs without github branches, these links need to be relative.

### CODE REVIEW

**Comment by abhay-krishna** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/aws/eks-anywhere/pull/364#" title="Author self-approved">TerryHowe</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=aws%2Feks-anywhere).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[docs/OWNERS](https://github.com/aws/eks-anywhere/blob/main/docs/OWNERS)~~ [TerryHowe]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by abhay-krishna** [comment]
@TerryHowe: The following test **failed**, say `/retest` to rerun all failed tests:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
eks-anywhere-docs-presubmit | 2cb1533f019245969988e93d098f04952b78c657 | [link](https://prow.eks.amazonaws.com/view/s3/prowpresubmitsdataclusterstack-prowbucket7c73355c-vfwwxd2eb4gp/pr-logs/pull/aws_eks-anywhere/364/eks-anywhere-docs-presubmit/1443968298004779008) | `/test eks-anywhere-docs-presubmit`

[Full PR test history](https://prow.eks.amazonaws.com/pr-history?org=aws&repo=eks-anywhere&pr=364). [Your PR dashboard](https://prow.eks.amazonaws.com/pr?query=is:pr+state:open+author:TerryHowe).

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>

File: docs/content/en/docs/tasks/troubleshoot/_troubleshooting.md
Comment (vivek-koppuru) [inline docs/content/en/docs/tasks/troubleshoot/_troubleshooting.md:218]: Does this work? I found out that I needed to change the content shortcode to use `Page` instead of `Site`

Comment (TerryHowe) [reply docs/content/en/docs/tasks/troubleshoot/_troubleshooting.md:218]: Well, it stopped complaining, but I noticed it doesn't render right

Comment (TerryHowe) [reply docs/content/en/docs/tasks/troubleshoot/_troubleshooting.md:218]: fixed

**Review by vivek-koppuru — APPROVED** [review]
/lgtm

**Comment by abhay-krishna** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/aws/eks-anywhere/pull/364#" title="Author self-approved">TerryHowe</a>*, *<a href="https://github.com/aws/eks-anywhere/pull/364#pullrequestreview-769345313" title="LGTM">vivek-koppuru</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=aws%2Feks-anywhere).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[docs/OWNERS](https://github.com/aws/eks-anywhere/blob/main/docs/OWNERS)~~ [TerryHowe,vivek-koppuru]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**
